default/techs.txt

```diff
@@ -2276,7 +2291,7 @@ Tech
     researchturns = 8
     prerequisites = [
         "SHP_MULTICELL_CAST"
-        "GRO_LIFECYCLE_MAN"
+        "SHP_MONSTER_MATURATION"
```

Comment:
The new tech was placed between GRO_LIFECYCLE_MAN and SHP_ENDOCRINE_SYSTEMS, because there was a long prequesite line between them, so SHP_MONSTER_MATURATION just fills the gap between the two.
But better wait for Dilvish to comment.